### PR TITLE
Plan construct in ConstructGraphPlanner instead of ConstructGraph operator

### PR DESCRIPTION
**Pull request by pstutz**

### CODE REVIEW

File: okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/operators/RelationalOperator.scala [hunk 1]
```diff
@@ -212,7 +212,7 @@ final case class Alias[T <: Table[T]](
   override lazy val header: RecordHeader = in.header.withAlias(aliases: _*)
 }
 
-final case class Add[T <: Table[T]](in: RelationalOperator[T], expr: Expr) extends RelationalOperator[T] {
+final case class Project[T <: Table[T]](in: RelationalOperator[T], expr: Expr) extends RelationalOperator[T] {
```
Comment (s1ck) [inline okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/operators/RelationalOperator.scala]: This is not performing a projection. It just adds a column to the underlying table.

File: okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/operators/RelationalOperator.scala [hunk 2]
```diff
@@ -238,15 +238,15 @@ final case class Add[T <: Table[T]](in: RelationalOperator[T], expr: Expr) exten
   }
 }
 
-final case class AddInto[T <: Table[T]](
+final case class ProjectInto[T <: Table[T]](
```
Comment (s1ck) [inline okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/operators/RelationalOperator.scala]: This is not performing a projection. It just adds a column to the underlying table.

File: okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/planning/ConstructGraphPlanner.scala [hunk 3]
```diff
@@ -293,66 +249,56 @@ final case class ConstructGraph[T <: Table[T]](
...
   }
-}
 
-case class RetagVariable[T <: Table[T]](
-  in: RelationalOperator[T],
-  v: Var,
-  replacements: Map[Int, Int]
-) extends RelationalOperator[T] {
+  /**
+    *org.apache.spark.sql.functions$#monotonically_increasing_id()
```
Comment (s1ck) [inline okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/planning/ConstructGraphPlanner.scala]: Remove or edit documentation

File: okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/planning/ConstructGraphPlanner.scala [hunk 4]
```diff
@@ -293,66 +249,56 @@ final case class ConstructGraph[T <: Table[T]](
...
-  valueExpr: Expr
-)
-  (implicit context: RelationalRuntimeContext[T]) extends RelationalOperator[T] {
+  override def returnItems: Option[Seq[Var]] = None
 
-  private lazy val existingPropertyExpressionsForKey = in.header.propertiesFor(v).collect {
-    case p@Property(_, PropertyKey(name)) if name == propertyExpr.key.name => p
+  override lazy val graph: RelationalCypherGraph[T] = {
+    // Register constructed graph in context
+    context.constructedGraphCatalog += (construct.name -> constructedGraph)
```
Comment (s1ck) [inline okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/planning/ConstructGraphPlanner.scala]: Should this happen during construction of `ConstructGraph`, i.e. outside of `val graph`?

File: okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/planning/RelationalPlanner.scala [hunk 5]
```diff
@@ -327,7 +347,7 @@ object RelationalPlanner {
         case _ => Set.empty
       }
       val withTrueLabels = trueLabels.foldLeft(withDroppedExpressions: RelationalOperator[T]) {
-        case (currentOp, label) => relational.AddInto(currentOp, TrueLit, HasLabel(targetVar, Label(label))(CTBoolean))
+        case (currentOp, label) => relational.ProjectInto(currentOp, TrueLit, HasLabel(targetVar, Label(label))(CTBoolean))
```
Comment (s1ck) [inline okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/planning/RelationalPlanner.scala]: this and the following lines could use the convenience methods defined above

File: okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/operators/RelationalOperator.scala (hunk 1 above)
Comment (Mats-SX) [reply okapi-relational/src/main/scala/org/opencypher/okapi/relational/impl/operators/RelationalOperator.scala]: What is the difference between a 'projection' and the operation of making another field available with a value given by an expression?

**Pull request merged**